src/api/chrisapiclient.ts

```diff
@@ -17,15 +17,15 @@ const AUTH_TOKEN_KEY = 'AUTH_TOKEN';
 class ChrisAPIClient {
 
   private static client: Client;
-  private static tokenUnauthorized: boolean;
+  private static isTokenAuthorized: boolean;
 
   static getClient(): Client {
-    if (!this.client || this.tokenUnauthorized) {
+    if (!this.client || !this.isTokenAuthorized) {
```

Comment:
how come there is a negation `!` here, when it was not used previously?

Reply:
The variable `tokenUnauthorized` was renamed to `isTokenAuthorized`, and so we needed to negate it.